packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/template.html

```diff
@@ -9,7 +8,7 @@ <h2 data-translate="octavia_load_balancer_pools_title"></h2>
         <span
             data-translate="octavia_load_balancer_pools_description_part_2"
         ></span>
-        <a href="{{ $ctrl.link }}" target="_blank" rel="noopener">
+        <a href="" data-ng-click="$ctrl.goToDiscoverOptions()">
```

Comment:
To handle the tracking please use the attributes track-on, track-name and track-type